src/black/__init__.py

```diff
@@ -498,6 +514,25 @@ def target_version_option_callback(
     callback=read_pyproject_toml,
     help="Read configuration from FILE path.",
 )
+@click.option(
+    "--report-type",
+    type=click.Choice([v.name.lower() for v in ReportType]),
+    callback=report_option_callback,
+    default=DEFAULT_REPORT,
+    multiple=False,
+    help=(
+        "Option if the generated Report should be to the Console or a Junit XML File.\n"
+        "[default : console]"
+    ),
+)
+@click.option(
```

Comment:
Not sure we need two options. We can just make it so if `--junit-xml` is given, we output XML.

Reply:
Maybe you are right.
My intention was if there will be more reports type in the future this could be a selector.